skfda/misc/scoring.py

```diff
@@ -1025,6 +1059,132 @@ def _msle_func(x: EvalPointsType) -> NDArrayFloat:  # noqa: WPS430
 
     return _multioutput_score_basis(y_true, multioutput, _msle_func)
 
+@overload
+def root_mean_squared_log_error(
```

Comment:
Shouldn't `root_mean_squared_error` also have overloads to take multioutput into account?